packages/components/icons/src/inline-svg.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Props } from './templates/icon.styles';
+
+import React from 'react';
+import DOMPurify from 'dompurify';
+import SVG from 'react-inlinesvg';
+import { useTheme } from '@emotion/react';
+import { getIconStyles } from './templates/icon.styles';
+
+type InlineSvgProps = Props & {
+  data: string;
+};
+
+const InlineSvg = (props: InlineSvgProps) => {
+  const theme = useTheme();
+  const sanitized = DOMPurify.sanitize(props.data, {
+    USE_PROFILES: { svg: true },
+    FORBID_ATTR: [
+      // To avoid injection by using `style="filter:url(\"data:image/svg+xml,<svg`
+      'style',
+    ],
+  });
```

Comment:
Here we sanitize now. I tested many different combinations (see `fixtures/xss.js`) and no "alert" popped up.